ui/app/pages/permissions-connect/choose-account/choose-account.component.js

```diff
@@ -76,31 +125,67 @@ export default class ChooseAccount extends Component {
     )
   }
 
+  renderAccountsListHeader () {
+    const { t } = this.context
+    const { selectNewAccountViaModal } = this.props
+    return (
+      <div className="permissions-connect-choose-account__accounts-list-header">
+        <div className="permissions-connect-choose-account__select-all">
+          <CheckBox
+            className="permissions-connect-choose-account__header-check-box"
+            checked={this.allAreSelected()}
+            onClick={() => (this.allAreSelected() ? this.deSelectAll() : this.selectAll())}
+          />
+          <div className="permissions-connect-choose-account__text-grey">{ this.context.t('selectAll') }</div>
+          <Tooltip
+            position="bottom"
+            html={(
+              <div style={{ width: 129, height: 112, padding: 4 }}>
+                {t('selectingAllWillAllow')}
+              </div>
+            )}
+          >
+            <i className="fa fa-info-circle" />
+          </Tooltip>
+        </div>
+        <div
+          className="permissions-connect-choose-account__text-blue"
+          onClick={() => selectNewAccountViaModal(this.handleAccountClick.bind(this))}
+        >
+          { this.context.t('newAccount') }
+        </div>
+      </div>
+    )
+  }
+
   render () {
-    const { originName, selectNewAccountViaModal, permissionsRequestId, cancelPermissionsRequest } = this.props
+    const { originName, selectAccounts, permissionsRequestId, cancelPermissionsRequest } = this.props
+    const { selectedAccounts } = this.state
     const { t } = this.context
     return (
       <div className="permissions-connect-choose-account">
         <div className="permissions-connect-choose-account__title">
-          { t('chooseAnAcount') }
+          { t('selectYourAccounts') }
```

Comment:
There should be an account icon shown above here

Reply:
this issue went away with aafa179f0